packages/opc-header/src/opc-header.scss

```diff
@@ -0,0 +1,36 @@
+//@import "../../opc-styles/opc-styles.scss";
```

Comment:
can you use the opc-style.scss
and use the fonts available in the in it.

Reply:
Can you plz give me more clarity on this, i already using font.

Reply:
@anjneeksharma I have added a suggestion here https://github.com/1-Platform/op-components/pull/11#discussion_r464865138

Should be helpful